Depth adjuster crashing after OpenCV error

When I launched `rocon_gocart`:

```
OpenCV Error: Assertion failed (0 <= roi.x && 0 <= roi.width && roi.x + roi.width <= m.cols && 0 <= roi.y && 0 <= roi.height && roi.y + roi.height <= m.rows) in Mat, file /build/buildd/opencv-2.4.8+dfsg1/modules/core/src/matrix.cpp, line 323
terminate called after throwing an instance of 'cv::Exception'
  what():  /build/buildd/opencv-2.4.8+dfsg1/modules/core/src/matrix.cpp:323: error: (-215) 0 <= roi.x && 0 <= roi.width && roi.x + roi.width <= m.cols && 0 <= roi.y && 0 <= roi.height && roi.y + roi.height <= m.rows in function Mat

[...]
[FATAL] [1453893840.808152920]: Service call failed!
[gopher/sensor_3d_long_range/sensor_3d_long_range_nodelet_manager-53] process has died [pid 26227, exit code -6, cmd /opt/ros/indigo/lib/nodelet/nodelet manager __name:=sensor_3d_long_range_nodelet_manager __log:=/home/yujin/.ros/log/71bab26e-c4e8-11e5-a9d1-d0509978a4ea/gopher-sensor_3d_long_range-sensor_3d_long_range_nodelet_manager-53.log].
log file: /home/yujin/.ros/log/71bab26e-c4e8-11e5-a9d1-d0509978a4ea/gopher-sensor_3d_long_range-sensor_3d_long_range_nodelet_manager-53*.log
[gopher/sensor_3d_long_range/depth_adjuster-56] process has died [pid 26267, exit code 255, cmd /opt/ros/indigo/lib/nodelet/nodelet load depth_calibration/depth_adjuster sensor_3d_long_range_nodelet_manager input_depth_raw:=uncalibrated_depth/image_raw output_depth_raw:=depth/image_raw input_camera_info:=uncalibrated_depth/camera_info output_camera_info_relay:=depth/camera_info __name:=depth_adjuster __log:=/home/yujin/.ros/log/71bab26e-c4e8-11e5-a9d1-d0509978a4ea/gopher-sensor_3d_long_range-depth_adjuster-56.log].
log file: /home/yujin/.ros/log/71bab26e-c4e8-11e5-a9d1-d0509978a4ea/gopher-sensor_3d_long_range-depth_adjuster-56*.log
```

Looks like it's related to the depth adjuster.

There are several seemingly helpful Q&As related to this error:

* http://answers.opencv.org/question/46506/opencv-error-assertion-failed/
* http://answers.opencv.org/question/21374/possible-solutions-for-opencv-assertion-error-problem/

Maybe there error is triggered somewhere around here: https://github.com/yujinrobot/depth_calibration/blob/indigo-devel/src/depth_adjuster.cpp#L277-L281

--- a/src/depth_adjuster.cpp
+++ b/src/depth_adjuster.cpp
@@ -12,13 +12,12 @@ void DepthAdjuster::onInit()
 {
   ros::NodeHandle& nh = getNodeHandle();
 
-  sub_depth_raw_ = nh.subscribe<sensor_msgs::Image>("input_depth_raw", 1, &DepthAdjuster::apply_calibration_cb, this);
   pub_calibrated_depth_raw_ = nh.advertise<sensor_msgs::Image>("output_depth_raw", 1);
   pub_inlier_ = nh.advertise<sensor_msgs::Image>("inlier", 1);
 
+  pub_camera_info_relay_ = nh.advertise<sensor_msgs::CameraInfo>("output_camera_info_relay", 1);
   sub_camera_info_ = nh.subscribe<sensor_msgs::CameraInfo>("input_camera_info", 1, &DepthAdjuster::relay_camera_info,
                                                            this);
-  pub_camera_info_relay_ = nh.advertise<sensor_msgs::CameraInfo>("output_camera_info_relay", 1);
 
   ros::NodeHandle& private_nh = getPrivateNodeHandle();
   bool enable;
@@ -45,6 +44,8 @@ void DepthAdjuster::onInit()
   private_nh.param("border_percentage_bottom", border_percentage_bottom_, 0.0);
   private_nh.param("border_percentage_left", border_percentage_left_, 0.0);
   private_nh.param("border_percentage_right", border_percentage_right_, 0.0);
+
+  sub_depth_raw_ = nh.subscribe<sensor_msgs::Image>("input_depth_raw", 1, &DepthAdjuster::apply_calibration_cb, this);
 }
 
 void DepthAdjuster::load_calibration(std::string file_path)

